workers/unity/Packages/com.improbable.gdk.playerlifecycle/PlayerLifecycleHelper.cs

```diff
@@ -21,6 +23,43 @@ public static class PlayerLifecycleHelper
             template.AddComponent(owningComponent, serverAccess);
         }
 
+        public static byte[] SerializeParams<T>(T playerCreationArguments)
+        {
+            try
+            {
+                using (var memoryStream = new MemoryStream())
+                {
+                    var binaryFormatter = new BinaryFormatter();
+                    binaryFormatter.Serialize(memoryStream, playerCreationArguments);
+                    return memoryStream.ToArray();
+                }
+            }
+            catch
+            {
+                UnityEngine.Debug.LogWarning("Unable to serialize player creation arguments.");
```

Comment:
This hides the exception

Reply:
yeah I fixed this locally but waiting to push until I incorporate PR feedback